openc3/lib/openc3/top_level.rb

```diff
@@ -565,3 +565,31 @@ def self.close_socket(socket)
     end
   end
 end
+
+# The following code makes most older COSMOS 5 plugins still work with OpenC3
+# New plugins should only use openc3 paths and module OpenC3
+unless ENV['OPENC3_NO_COSMOS_COMPATIBILITY']
```

Comment:
Should this go in the .env file defined as 1 and then people can remove if they need this compatibility?

Reply:
I think so in the future.   For now during the transition we should probably leave it in don't you think?   Or not?

Reply:
The .env file doesn't support comments or does it? I was going to suggest commenting it out but I don't think it supports that. Yeah I agree it should be enabled by default for now.